kong/clustering/rpc/concentrator.lua

```diff
@@ -90,6 +93,67 @@ local function enqueue_notifications(notifications, notifications_queue)
 end
 
 
+function _M:process_one_response(payload)
+  assert(payload.jsonrpc == jsonrpc.VERSION)
```

Comment:
We need to handle errors here. It could be an error of the opponent thus we should not fail in this case.

Reply:
Now this PR just refactored and moved the original code, kept the logic, we could change it later.